src/OrchardCore/OrchardCore.Abstractions/Extensions/OrchardHelperExtensions.cs

```diff
@@ -0,0 +1,21 @@
+using System.Net;
+
+namespace OrchardCore;
+
+public static class OrchardHelperExtensions
+{
+    public static IPAddress GetClientIpAddress(this IOrchardHelper orchardHelper)
```

Comment:
I agree with the others this could be an extension for `HttpContext` not `IOrchardHelper`